src/indicators/cholera/README.md

```diff
@@ -35,9 +35,9 @@ date of the outbreak. The end date is the date that the flag was raised.
 
 ## Output datasets
  
-`flags_cholera.gdsheets`: flagging dataset.
+`flags.parquet`: flagging dataset.
 
-`raw_cholera.gdsheets`: raw cholera output from the CERF scraping.
+`raw.parquet`: raw cholera output from the CERF scraping.
```

Comment:
`raw.parquet`: raw cholera output from CERF scraping (downloaded as .csv and converted to .parquet)